CIP-0019/README.md

```diff
@@ -12,19 +12,43 @@ License: CC-BY-4.0
 
 # Abstract
 
-This specification describes the binary structure of addresses in Cardano, covering both addresses introduced in the Shelley era and the legacy format from the Byron era.
+This specification describes the structure of addresses in Cardano, covering both addresses introduced in the Shelley era and the legacy format from the Byron era.
 
 # Motivation
 
 Document design choices for posterity. Most applications interacting with the Cardano blockchain will likely not have any need for this level of details, however, some might. This CIP is meant to capture this knowledge. 
 
 # Specification
 
-## Overview
+## Introduction
 
-In Cardano, addresses are comprised of two parts, a **header** and a **payload**. Depending on the header, the interpretation of the payload varies. 
+In Cardano, an address is a **sequence of bytes** that conforms to a particular format, which we describe below.
 
-In the header, bits [7;4] indicate the type of addresses being used; we'll call these four bits the **header type**. The remaining four bits [3;0] are either unused or refer to what we'll call the **network tag**. There are currently 11 types of addresses in Cardano which we'll divide into three categories: [Shelley addresses], [stake addresses], and [Byron addresses]. 
+However, users will typically come into contact with addresses only after these addresses have been **encoded** into sequences of human-readable characters. In Cardano, the [Bech32][] and [Base58][] encodings are used to encode addresses, as opposed to standard hexadecimal notation (Base16, example `0x8A7B`). These encoded sequence of characters have to be distinguished from the byte sequences that they encode, but lay users will (and should) perceive the encoded form as "the" address.
+
+## User-facing Encoding 
+
+By convention, **Shelley** and stake addresses are encoded using **[Bech32][]** with human-readable prefixes defined in [CIP-0005][]. The most common prefix is `addr`, representing an address on mainnet. Bech32 is the preferred encoding, as its built-in error detection may protect users against accidental misspellings or truncations. The encoding in Cardano does not strictly follow the Bech32 standard, but the only difference is that the character sequence may be longer than the 90 character length limit.
```

Comment:
It does strictly follow the Bech32 standard, it simply makes the error detection a little less efficient. As per the specification:

> Even though the chosen code performs reasonably well up to 1023 characters, other designs are preferable for lengths above 89 characters (excluding the separator).

Reply:
Hm. Unfortunately, the standard (BIP-0173) specifically says that "A Bech32[2] string is at most 90 characters long and consists of: […]". 😅
But I can word it in a more positive manner and avoid presenting as a failure to comply with the standard.